test/conversion_test.go

```diff
@@ -0,0 +1,879 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"github.com/tektoncd/pipeline/test/parse"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	knativetest "knative.dev/pkg/test"
+	"knative.dev/pkg/test/helpers"
+)
+
+var (
+	// supportedConversionFeatureGates = map[string]string{
+	// 	"enable-tekton-oci-bundles": "true",
+	// }
```

Comment:
I'm thinking if we want to keep this conversion test in both `pull-tekton-pipeline-build-tests` and `pull-tekton-pipeline-alpha-integration-tests`, which means we may need to write different test cases for different feature-gate values.

Reply:
An example: 

https://github.com/tektoncd/pipeline/blob/1b941d31d7a47c8710cc19bc3b161ce265aa96dc/test/retry_test.go#L50

https://github.com/tektoncd/pipeline/blob/1b941d31d7a47c8710cc19bc3b161ce265aa96dc/test/featureflags.go#L93-L103